src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -69,6 +69,13 @@ def create_connectedk8s(cmd, client, resource_group_name, cluster_name, https_pr
     # Escaping comma, forward slash present in no proxy urls, needed for helm params.
     no_proxy = escape_proxy_settings(no_proxy)
 
+    # Checking whether tokenbasedonboarding has been opted from the values file.
+    token_based_onboarding = False
+    values_file = os.getenv('HELMVALUESPATH')
+    if (values_file is not None) and (os.path.isfile(values_file)):
```

Comment:
What happens if the file is not present?

Reply:
If not present, normal onboarding will be done.